studio/src/studio.js

```diff
@@ -55,7 +55,7 @@ class MasStudio extends LitElement {
         this.addEventListener(EVENT_LOAD_END, () => this.requestUpdate());
         this.addEventListener(EVENT_CHANGE, () => {
             if (!this.fragment) this.showEditorPanel = false;
-            this.requestUpdate();
+            else this.requestUpdate();
```

Comment:
Because when the fragment is copied, the new fragment is set as the currently in edit one. So I have to refresh the studio. If the fragment is null ok, I can just close the editor, but otherwise I have to request an update.

Reply:
Got it, just wanted to understand, thanks.